corehq/warehouse/transforms/sql/app_status_fact.sql

```diff
@@ -0,0 +1,45 @@
+SELECT * FROM
```

Comment:
ok. before this goes live i think we're really going to want to split this into intermediate tables - we shouldn't ever join results from subqueries